frontend/src/__tests__/cypress/cypress/support/e2e.ts

```diff
@@ -28,6 +28,14 @@ Cypress.Keyboard.defaults({
   keystrokeDelay: 0,
 });
 
+//TODO: Delete this once https://issues.redhat.com/browse/RHOAIENG-10719 is fixed
+Cypress.on('uncaught:exception', () => false);
```

Comment:
Can we move this to the individual test case instead of globally?
Use `cy.on('uncaught:exception', () => false);` when in a single test case.

Reply:
I don't really see the perks of moving it from globally, as the next test could also fail due to this uncaught exception and then, the line should be added to the new test too.
Done here: [2b126d1](https://github.com/opendatahub-io/odh-dashboard/pull/3028/commits/2b126d1c2b1db16307a66a2e75cfafcd47ddcb68)